app/Models/Task.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Task extends Model
+{
+    protected $fillable = ['title','body'];
+    use HasFactory;
+    public function team(){
+        return $this->belongsTo(Team::class);
+    }
```

Comment:
use は一番上に固めておく方が良さそうです。

```suggestion
    use HasFactory;

    protected $fillable = ['title','body'];
    public function team(){
        return $this->belongsTo(Team::class);
    }
```